Quickstart/00-Starter-Seed/WebApi/WebApi/packages.config

```diff
@@ -3,15 +3,17 @@
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net452" />
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net452" />
   <package id="Microsoft.AspNet.WebApi.Owin" version="5.2.3" targetFramework="net452" />
-  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.1" targetFramework="net452" />
-  <package id="Microsoft.Net.Compilers" version="1.3.2" targetFramework="net452" developmentDependency="true" />
-  <package id="Microsoft.Owin" version="3.0.1" targetFramework="net452" />
-  <package id="Microsoft.Owin.Host.SystemWeb" version="3.0.1" targetFramework="net452" />
-  <package id="Microsoft.Owin.Security" version="3.0.1" targetFramework="net452" />
-  <package id="Microsoft.Owin.Security.ActiveDirectory" version="3.0.1" targetFramework="net452" />
-  <package id="Microsoft.Owin.Security.Jwt" version="3.0.1" targetFramework="net452" />
-  <package id="Microsoft.Owin.Security.OAuth" version="3.0.1" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
+  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.4" targetFramework="net452" />
+  <package id="Microsoft.IdentityModel.Logging" version="1.1.3" targetFramework="net452" />
+  <package id="Microsoft.IdentityModel.Tokens" version="5.1.3" targetFramework="net452" />
+  <package id="Microsoft.Net.Compilers" version="2.1.0" targetFramework="net452" developmentDependency="true" />
+  <package id="Microsoft.Owin" version="3.1.0" targetFramework="net452" />
+  <package id="Microsoft.Owin.Host.SystemWeb" version="3.1.0" targetFramework="net452" />
+  <package id="Microsoft.Owin.Security" version="3.1.0" targetFramework="net452" />
+  <package id="Microsoft.Owin.Security.ActiveDirectory" version="3.1.0" targetFramework="net452" />
+  <package id="Microsoft.Owin.Security.Jwt" version="3.1.0" targetFramework="net452" />
+  <package id="Microsoft.Owin.Security.OAuth" version="3.1.0" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="10.0.2" targetFramework="net452" />
   <package id="Owin" version="1.0" targetFramework="net452" />
-  <package id="System.IdentityModel.Tokens.Jwt" version="4.0.2.206221351" targetFramework="net452" />
+  <package id="System.IdentityModel.Tokens.Jwt" version="5.1.3" targetFramework="net452" />
```

Comment:
@Amialc Have you tested this with 5.1.3? I seem to recall that 5.x gave errors before...